Makefile

```diff
@@ -5,7 +5,8 @@
 GO_LINES_IGNORED_DIRS=contracts
 GO_PACKAGES=./chainio/... ./crypto/... ./logging/... \
 	./types/... ./utils/... ./signer/... ./cmd/... \
-	./signerv2/...
+	./signerv2/... ./aws/... ./internal/... ./metrics/... \
+	./nodeapi/... ./cmd/... ./services/... ./testutils/...
```

Comment:
Is there a way to specify directories to ignore (like .gitignore) instead of ones we want to add? Otherwise we'll keep forgetting to add them here when we add new packages.

Reply:
yea I checked - this library doesn't do that and requires all pkgs :( . I can write some script to automatically find it I think